script/caffe2intel-i8/finalize_train_val_prototxt.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2018 cTuning foundation.
+# See CK COPYRIGHT.txt for copyright details.
+#
+# SPDX-License-Identifier: BSD-3-Clause.
+# See CK LICENSE.txt for licensing details.
+#
+
+import argparse
+import os
+
+from caffe.proto import caffe_pb2
+from google.protobuf import text_format  
+
+if __name__ == '__main__':
+  parser = argparse.ArgumentParser()
+  parser.add_argument('-m', '--model', action='store', dest='MODEL')
+  params = parser.parse_args()
+
+  print('Postprocessing {} ...'.format(params.MODEL))
+
+  net = caffe_pb2.NetParameter()
+  with open(params.MODEL, 'r') as f:
+    text_format.Merge(f.read(), net)
+
+  # Adjust input layer
+  # Currently it's supposed that there is only one input layer
+  for layer in net.layer:
+    if layer.name == 'data':
+      layer.transform_param.mean_file = '$#val_mean#$'
+      layer.data_param.source = '$#val_lmdb#$'
+      layer.data_param.batch_size = 0
+
+  # Serialize prototxt
+  txt = text_format.MessageToString(net)
+
+  # We cant insert strings into integer field using caffe_pb2 
```

Comment:
Same as in `finalize_deploy_prototxt.py`.

Reply:
Fixed.